exporter/otlpexporter/otlp.go

```diff
@@ -50,12 +51,17 @@ func newExporter(cfg config.Exporter) (*exporter, error) {
 
 	e := &exporter{}
 	e.config = oCfg
-	w, err := newGrpcSender(oCfg)
+	return e, nil
+}
+
+// start actually creates the gRPC connection. The client construction is deferred till this point as this
+// is the only place we get hold of Extensions which are required to construct auth round tripper.
+func (e *exporter) start(_ context.Context, _ component.Host) (err error) {
+	e.w, err = newGrpcSender(e.config)
 	if err != nil {
-		return nil, err
+		return err
 	}
```

Comment:
No need to have the if here, just return err :)

Reply:
ah good stuff - updating the zipkin one as well.